components/ui/partner-card.js

```diff
@@ -1,41 +1,34 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-class PartnerCard extends React.Component {
-  static propTypes = {
-    logo: PropTypes.string,
-    title: PropTypes.string,
-    description: PropTypes.string,
-    url: PropTypes.string,
-    maxWidth: PropTypes.string
-  };
+const PartnerCard = (props) => {
+  const { logo, name, website, description } = props;
 
-  static defaultProps = {
-    logo: '',
-    title: '',
-    description: '',
-    url: '',
-    maxWidth: ''
-  };
-
-  render() {
-    const { logo, title, description, url, maxWidth } = this.props;
-    return (
-      <div className="c-partner-card">
-        <div className="partner-card-logo">
-          <a rel="noopener noreferrer" target="_blank" href={url}>
-            <h3>{title}</h3>
-            <img style={{ maxWidth }} alt={title} src={logo} />
-          </a>
-        </div>
-        <div className="partner-card-content">
-          <p>
-            {description}
-          </p>
-        </div>
+  return (
+    <div className="c-partner-card">
+      <div className="partner-card-logo-container">
+        <a rel="noopener noreferrer" target="_blank" href={website}>
+          <h3 className="partner-card-title">{name}</h3>
+          <picture>
```

Comment:
![image](https://user-images.githubusercontent.com/8928965/35148887-4f0b0a82-fd14-11e7-92c4-6514f57211a5.png)

Reply:
I was thinking that the `<picture>` element gracefully degrades to the `<img>` element in unsupported browsers 🤔.

Reply:
Check it and sent the confirmation. In that case, BRAVO

Reply:
This was old w3c draft spec, can't say anything explicitly in latest, would need to confirm by testing on an older browser:
![image](https://user-images.githubusercontent.com/8505382/35153083-544cf62a-fd25-11e7-89b2-826cdefc5ca8.png)

From https://developers.google.com/web/ilt/pwa/lab-responsive-images: 
![image](https://user-images.githubusercontent.com/8505382/35153238-edaa2cd4-fd25-11e7-8ffc-33024082fafc.png)

Reply:
👍